artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py

```diff
@@ -425,4 +426,6 @@
     "Hestia Control Panel login was detected.": "wykryto panel logowania narzędzia Hestia Control Panel.",
     "Kibana login panel was detected.": "wykryto panel logowania narzędzia Kibana.",
     "Check for the existence of the ArcGIS Token Service on an ArcGIS server.": "wykryto narzędzie ArcGIS Token Service.",
+    "The jUDDI (Java Universal Description, Discovery and Integration) Registry is a core component of the JBoss Enterprise SOA Platform. It is the product's default service registry and comes included as part of the product. In it are stored the addresses (end-point references) of all the services connected to the Enterprise Service Bus. It was implemented in JAXR and conforms to the UDDI specifications.": "wykryto narzędzie jUDDI (Java Universal Description, Discovery and Integration) Registry.",
+    'Check for the existence of the "/arcgis/rest/services" path on an ArcGIS server.': "wykryto API narzędzia ArcGIS.",
```

Comment:
did we already detect this, but didn't have the translations?

Reply:
yes - I add translations only after a template actually finds something